packages/passport-interface/src/api/makeRequest.ts

```diff
@@ -0,0 +1,119 @@
+import { getErrorMessage } from "@pcd/util";
+import urlJoin from "url-join";
+import { APIResult, GetResultValue, ResultMapper } from "./apiResult";
+import { POST } from "./constants";
+
+/**
+ * Wrapper of {@link httpRequest} that sends a GET request.
+ */
+export async function httpGet<T extends APIResult<unknown, unknown>>(
+  url: string,
+  opts: ResultMapper<T>,
+  query?: object
```

Comment:
question: since we're casting it later, why not just explicitly type it here?
```suggestion
  query?: Record<string, string>
```

Reply:
unfortunately the type checker won't let me do this